index.html

```diff
@@ -158,85 +158,36 @@ <h3 id="featured-project-title">Multi-Post Stories</h3>
-                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                        <ul class="project-languages">
-                            <li>html</li>
-                            <li>bootstrap</li>
-                            <li>Ruby</li>
-                        </ul>
-                    </div>
-                    <button onclick="location.href='https://mirouhml.github.io/portfolio/index.html'" class="project-button" type="button">See Project</button>
-                </li>
-                <li class="project">
-                    <div class="project-card-content">
-                        <h3 class="project-title">Profesional Art Printing Data</h3>
-                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                        <ul class="project-languages">
-                            <li>html</li>
-                            <li>bootstrap</li>
-                            <li>Ruby</li>
-                        </ul>
-                    </div>
-                    <button onclick="location.href='https://mirouhml.github.io/portfolio/index.html'" class="project-button" type="button">See Project</button>
-                </li>
-                <li class="project">
-                    <div class="project-card-content">
-                        <h3 class="project-title">Profesional Art Printing Data</h3>
-                        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                        <ul class="project-languages">
-                            <li>html</li>
-                            <li>bootstrap</li>
-                            <li>Ruby</li>
-                        </ul>
-                    </div>
-                    <div class="project-card-hover-content">
-                        <button onclick="location.href='https://mirouhml.github.io/portfolio/index.html'" class="project-button" type="button">See Project</button>
+                    <div class="popup-body">
+                        <img src="src/popup-image.svg" id="popup-image" alt="Project Image">
+                        <div class="popup-description">
+                            <p id="popup-description-text"></p>
+                            <div class="popup-buttons">
+                                <button id="live-button" class="featured-project-button" type="button">
+                                    See Live
+                                </button>
+                                <button id="source-button" class="featured-project-button" type="button">
+                                    See Source
+                                </button>
+                            </div>
```

Comment:
- [ ] The popup works but I suggest you work a bit on the responsiveness, it is not easy almost impossible to find the `X` button in small screens if the the browser is Zoomed to 100%, kindly adjust that behaviour so that your portfolio will be more accessible, kindly watch the following screencast to learn more about my concern, 

https://user-images.githubusercontent.com/56886064/151073404-d600521f-d55e-40e8-a313-8246435aed28.mp4

Reply:
Thank you for the detailed explanation, I've changed it to be better I hope this is okay.